### PR TITLE
NOTIF-381 Add correct admin mapping

### PR DESCRIPTION
**Pull request by bennetelli**

### CODE REVIEW

File: engine/src/test/java/com/redhat/cloud/notifications/recipients/rbac/RbacRecipientUsersProviderTest.java [hunk 1]
```diff
@@ -49,6 +52,59 @@
...
+
+        List<ITUserResponse> itUserResponses = createTestResponse("portal_download");
+
+        final List<User> users = testee.transformToUser(itUserResponses);
+
+        assertFalse(users.get(0).isAdmin());
+    }
+
+    private List<ITUserResponse> createTestResponse(String permissionCode) {
+        List<ITUserResponse> itUserResponses = new LinkedList<>();
```
Comment (gwenneg) [inline engine/src/test/java/com/redhat/cloud/notifications/recipients/rbac/RbacRecipientUsersProviderTest.java:78]: Please do not use `LinkedList` as the default implementation when you need to create a `List` instance.

Comment (bennetelli) [reply engine/src/test/java/com/redhat/cloud/notifications/recipients/rbac/RbacRecipientUsersProviderTest.java:78]: We build the second largest ecommerce shop in europe and didnt have any significant performance issues or improvements with one if this. But we can go with ArrayList if you want.

Comment (bennetelli) [reply engine/src/test/java/com/redhat/cloud/notifications/recipients/rbac/RbacRecipientUsersProviderTest.java:78]: ok :)

**Review by gwenneg — APPROVED** [review]
The tests still contain `LinkedList` instances but it's only a minor issue, so LGTM

**Comment by bennetelli** [comment]
> The tests still contain `LinkedList` instances but it's only a minor issue, so LGTM

with the latest commit I replaced every LinkedList with ArrayList.

**Comment by gwenneg** [comment]
Thanks!

**Pull request merged**